test/test_ops.py

```diff
@@ -263,9 +263,6 @@ def test_noncontiguous_samples(self, device, dtype, op):
             if not test_grad:
                 continue
 
-            expected = sample_input.output_process_fn_grad(expected)
-            actual = sample_input.output_process_fn_grad(actual)
-
```

Comment:
@mruberry, @Lezcano mentioned offline that these lines should probably not be removed